lib/data/user.ts

```
@@ -100,6 +101,7 @@ export const updateUserMailboxData = async (
   });
   if (updateError) throw updateError;
   if (!updatedUser) throw new Error("Failed to update user");
+  console.log("upsatedUser", updatedUser.user_metadata.mailboxAccess);
```

ESLint

[error] Unexpected console statement.

(no-console)